sass/layout/_sidebar-left.scss

```diff
@@ -257,6 +257,81 @@
         outline-style: none;
     }
 
+    .SidebarMenu {
```

Comment:
Maybe more of this should be done in a separate file? Similar to the widget code, you could have these styles next to the component that they're styling so that we can keep them apart from the older code that's in need of cleanup

Reply:
This file is purely for the new sidebar code, so it won't affect any of the older stuff.

I'm putting it here because the styles used here are from the spec for the sidebar, and we don't have a concrete plan of how we're fixing the menus yet. I assume some of this code will make it into the `Menu` component, but I don't think it makes sense to refactor any of this now without a plan.

Reply:
We could move this into the same folder as the rest of the `SidebarMenu` code at least even if some of it will be moved out to be with the proper menu components later on. It's more about making the styles easy to find and work with rather than refactoring the menus at this point.

Reply:
Can we do this as a later step perhaps? I'd like to first understand what needs to be refactored out, and since we're taking the steps to fix the menu sooner rather than later, I don't want to have to do my work twice.

Reply:
It's not a matter of major refactoring, it's just to move the newly added styles to some `.../sidebar_channel_menu/styles.scss` and import that in the component instead of adding it as part of the `sass` folder. Doing that doesn't change the names or anything, but it helps keep the CSS nearer to the code that should be affected by it.

Reply:
Is that a pattern we use often? I've never seen any CSS files outside the `sass` directory.

Reply:
We will move this out once we look into refactoring the menu.